src/vs/monaco.d.ts

```diff
@@ -7229,6 +7229,11 @@ declare namespace monaco.languages {
 		 * Can decrease the verbosity of the hover
 		 */
 		canDecreaseVerbosity?: boolean;
+		/**
+		 * Id of the extension that provided this hover.
+		 * Used to filter duplicate hovers from certain extensions.
+		 */
+		extensionId?: string;
```

Comment:
I think the compile issue @juliasilge was seeing is that I had Positron code fences around this, and the file was regenerated without them. Not great that we can't include fences here but it should be okay if it's generated. 

The same error was occurring on the Windows e2e build, so if that's fixed then we should be good to go.